misk-aws/src/main/kotlin/misk/jobqueue/sqs/ForSqsConsumer.kt

```diff
@@ -5,3 +5,6 @@ import javax.inject.Qualifier
 @Qualifier
 @Target(AnnotationTarget.FIELD, AnnotationTarget.FUNCTION, AnnotationTarget.VALUE_PARAMETER)
 internal annotation class ForSqsConsumer
+
+@Qualifier
+internal annotation class ForSqsReceiving
```

Comment:
Perhaps outside the scope of this PR but maybe we should also rename `ForSqsConsumer` to `ForSqsConsumption` so these two match.

Reply:
I would prefer doing that change in a different PR

Reply:
yea it's pretty confusing, but the idea with the internal impl was a consumer task (one per vm), can spawn many receiver threads. each receiver thread calls the SQS Receive api method.